numpy/core/memmap.py

```diff
@@ -22,7 +22,6 @@
 @set_module('numpy')
 class memmap(ndarray):
     """Create a memory-map to an array stored in a *binary* file on disk.
-
     Memory-mapped files are used for accessing small segments of large files
```

Comment:
```suggestion

    Memory-mapped files are used for accessing small segments of large files
```
Hah, github didn't like the empty line...